test/plugins/mergeStyles.10.svg

```diff
@@ -0,0 +1,15 @@
+Check whether plugin removes one empty <style> element that is also the only <style> element.
+
+===
+
+<svg id="test" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
+    <style>
+	  </style>
```

Comment:
A great job. There are a few tips:

The test case should include:
```html
  <style>/* test */</style>
  <style />
```

Reply:
@yisibl do you want to send PR?

Reply:
OK.

Reply:
@TrySound Do we now have an API to recognize CSS comments?

Reply:
The minify styles plugin can handle this before and after this plugin.

Reply:
What if we merge styles after minifying? I asked to move before first because there was unwanted newlines though @strarsis removed them and maybe it makes sense to merge after now.

Reply:
Yes, merging after minification is nice. The minifier removed all comments and only left the bare minimum.